skins/wmde19/src/components/layout/Footer.vue

```diff
@@ -1,11 +1,81 @@
 <template>
-	<div class="footer">
-		Footer
+	<div class="footer has-background-footer">
```

Comment:
The footer does not stick to the bottom of the page for big resolutions.
`margin-top: 80px` on `.footer` fixes that

Reply:
I fixed this with flex-boxes, should be good now.